source/common/listener_manager/active_tcp_socket.h

```diff
@@ -88,7 +91,7 @@ class ActiveTcpSocket : public Network::ListenerFilterManager,
   // The owner of this ActiveTcpSocket.
   ActiveStreamListenerBase& listener_;
   Network::ConnectionSocketPtr socket_;
-  const bool hand_off_restored_destination_connections_;
+  bool hand_off_restored_destination_connections_{};
```

Comment:
I know this predates you but can you comment? it's not obvious to me what the variable originally did or why or how